src/cltk/dependency/spacy_dep.py

```diff
@@ -31,34 +27,42 @@ class SpacyWrapper:
 
     def __init__(self,
                  language: str,
+                 spacy_language: spacy.Language = None,
```

Comment:
I think the naming is not entirely clear, especially since I don't think a large portion of spacy users actually knows that the object is of type `Language`.
In the spaCy docs they always have examples with the name `nlp`, and I think that would work better here too.

E.g.
```python
import spacy

nlp = spacy.load("en_core_web_sm")
doc = nlp("Hello eveyone nice to see you.")
```

Note that nowehre in this snippet of code is there any mention of `Language`, I think we should keep to their naming conventions.

Reply:
Ok for the `nlp` attribute name, it is even already used later in the code (`nlps` a class attribute).